README.rst

```diff
@@ -14,63 +14,82 @@
 NVIDIA Neural Modules: NeMo
 ===========================
 
-Neural Modules (NeMo) is a framework-agnostic toolkit for building AI applications powered by Neural Modules. Current support is for PyTorch framework.
+NeMo (**Ne**ural **Mo**dules) is a toolkit for creating AI applications built around **neural modules**, conceptual blocks of neural networks that take *typed* inputs and produce *typed* outputs. Such modules typically represent data layers, encoders, decoders, language models, loss functions, or methods of combining activations.
```

Comment:
The first phrase looks a strange when the actual README is rendered on GitHub (just navigate to "typo") branch to see. Maybe get rid of some bolding to fix it?